keyboards/numatreus/keymaps/hdbx/keymap.c

```diff
@@ -0,0 +1,237 @@
+// WindowsでJIS配列として認識しているときに、オシャレなUS配列用キーキャップを使うためのキーマップです。
+// Google日本語入力の場合、以下のキー設定を行うとLower,Raiseのタップでそれぞれ半角入力と全角入力を切り替えできるようになります。
+//  ・Henkan（入力文字なし/直接入力）に「IMEを有効化」を割り当て
+//  ・Muhenkan（入力文字なし/直接入力）に「IMEを無効化」を割り当て
+
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"       // qmk_firmware-master/quantum/keymap_extras/keymap_jp.h 日本語キーボード設定用
+#include <sendstring_jis.h>  // macro sendstring for jis keyboard マクロ文字列送信時に日本語キーボード設定での文字化け回避
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+    _QWERTY = 0,
+    _HDBX,
+    _LOWER,
+    _RAISE,
+    _GAME,
+    _ADJUST
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  HDBX,
+  MCR1,                // マクロ1
+  MCR2,                // マクロ2
+  MCR3,                // マクロ3
+  MCR4,                // マクロ4
+  MCR5,                // マクロ5
+  DYNAMIC_MACRO_RANGE, // ダイナミックマクロ
+  WN_SCLN              // タップでJISの「:」  シフトでJISの「;」 (Windows)
+};
+
+// Use Dynamic macro
+#include "dynamic_macro.h"
+
+// Fillers to make layering more clear
+#define LOWER   LT(_LOWER, KC_MHEN)    // タップで無変換     ホールドでLower
+#define RAISE   LT(_RAISE, KC_HENK)    // タップで変換       ホールドでRaise
+#define GUI_ESC GUI_T(KC_ESC)          // タップでESC        ホールドでGUI
+#define SFT_BS  SFT_T(KC_BSPC)         // タップでBackSpace  ホールドでSHIFT
+#define CTL_ENT CTL_T(KC_ENT)          // タップでEnter      ホールドでCTRL
+#define CTL_TAB CTL_T(KC_TAB)          // タップでTAB        ホールドでCTRL
+#define SFT_DEL SFT_T(KC_DEL)          // タップでDELETE     ホールドでSHIFT
+#define ALT_LBR ALT_T(JP_LBRC)         // タップで[          ホールドでALT
+#define SFT_RBR SFT_T(JP_RBRC)         // タップで]          ホールドでSHIFT
+#define SFT_SPC S(KC_SPC)              // Shift + Space
+#define WN_CAPS S(KC_CAPS)             // Caps Lock           (Windows)
+#define MPLAY1  DYN_MACRO_PLAY1        // ダイナミックマクロ1 
+#define MPLAY2  DYN_MACRO_PLAY2        // ダイナミックマクロ2
+#define MREC1   DYN_REC_START1         // ダイナミックマクロ1記録開始
+#define MREC2   DYN_REC_START2         // ダイナミックマクロ2記録開始
+#define MRSTOP  DYN_REC_STOP           // ダイナミックマクロ記録終了
```

Comment:
These aren't needed.  There are short form keycodes for these now: 

```
DM_REC1
DM_REC2
DM_PLY1
DM_PLY2
DM_RSTP
```

Reply:
It has changed.